cmd/utils.go

```diff
@@ -130,3 +130,23 @@ func buildS3Config(alias, urlStr string) (*Config, *probe.Error) {
 
 	return s3Config, nil
 }
+
+// isOlder returns true if the passed object is older than olderRef
+func isOlder(c *clientContent, olderRef int) bool {
+	now := time.Now().UTC()
+	objectAge := now.Sub(c.Time)
+	if objectAge < (time.Duration(olderRef) * Day) {
```

Comment:
The `if` is not required. you could just do `return objectAge < (time.Duration(olderRef) * Day)`